p2p/dhtdiscovery/discovery.go

```diff
@@ -292,3 +292,24 @@ func (d *Discovery) newDht(ctx context.Context, h host.Host, public, server bool
 	d.datastore = ds
 	return nil
 }
+
+func containsPeer(addrInfo []peer.AddrInfo, id peer.ID) bool {
```

Comment:
please move all of this to p2p/host.go

Reply:
Are you referring just to containsPeer or all the IsXXX functions as well?

Reply:
yes i meant all IsXXX

Reply:
Ok all moved